Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/shotgun.yml

```diff
@@ -48,8 +48,8 @@
         map: [ "enum.AmmoVisualLayers.Base" ]
   - type: CartridgeAmmo
     proto: PelletShotgunSlug
-    count: 4
-    spread: 5
+    count: 1
+    spread: 3
```

Comment:
this is literally just an unreliable pistol? It only shoots one bullet?

Reply:
Ideally I think shotgun slugs would somehow have their own spread configuration.

Reply:
Spread doesn't do anything if 1 is fired.